tests/test_model.py

```diff
@@ -19,3 +23,27 @@ def test_querying(create, tmpdir):
     assert len(set(db.todos())) == 9
     assert len(set(db.todos(lists='ab'))) == 6
     assert len(set(db.todos(lists='ab', location='a'))) == 2
+
+
+def test_retain_tz(tmpdir, runner, create, default_database):
+    create(
+        'ar.ics',
+        'SUMMARY:blah.ar\n'
+        'DUE;VALUE=DATE-TIME;TZID=ART:20160102T000000\n'
```

Comment:
@geier Thanks. That pretty much answers everything. Out of curiosity: is this a "standard" list, or pytz-specific?

@untitaker AR isn't mentioned in `all_timezones` either (and CET isn't in the list on that link).